### ISSUE TITLE
test run cinema vs vit

### ISSUE THREAD

**Issue by LevanBokeria** [alan-turing-institute/heartfm-data#18]
This issue is for running an initial test run, pretraining CineMA on M&M dataset and comparing it to pretraining a vanilla ViT on the same dataset.

We will run this on Baskerville, not expecting meaningful results but just an opportunity to play around with sending jobs and logging the right info in tensorboard and otherwise.

**Comment by LevanBokeria** [comment]
Current outstanding issues: 

- ViT: the reconstructed images seem to also reconstruct unmasked patches. Should limit reconstruction to masked patches only as in CineMA.
- ViT: learning rate schedule is likely suboptimal. I think transformers benefit from warmup runs, and our ViT training doesn't currently have them.
- ViT: current training does not use gradient clipping or gradient checkpointing. Likely both should be implemented. 
- Overall: improve the data preprocessing and data loading, e.g. how frames are handled. 
- Overall: test different batch sizes to examine memory consumption.
- Overall: run 50 epochs for both models.
